index.js

```diff
@@ -76,22 +76,27 @@ program
 
 
 const checkFileExistence = (cb) => {
-    fs.access(config.CONF_FILE, (error) => {
+    return fs.access(config.CONF_FILE, (error) => {
```

Comment:
I don't think you need `return` here, it's a bit confusing

Reply:
It's just returning function - to make sure it returns - can be used without `return` but in the future, if we add more things, we need to make sure that this is the final call to return.

Reply:
Done!